english/explore-outside/journey-to-flaming-tower/jellybus/jellybus.md

```diff
@@ -3,6 +3,8 @@ At the nearby jellybus stop you see that the next jellybus will be arriving in
 
 Do you:
 
+[Keep walking because you're too socially awkward to sit near a stranger for any length of time.](too-socially-awkward/too-socially-awkward.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.